trunk/SUAVE/Analyses/Mission/Segments/Cruise/Constant_Speed_Constant_Altitude.py

```diff
@@ -45,7 +45,7 @@ def __defaults__(self):
         # initials and unknowns
         ones_row = self.state.ones_row
         self.state.unknowns.throttle   = ones_row(1) * 0.5
-        self.state.unknowns.body_angle = ones_row(1) * 0.0
+        self.state.unknowns.body_angle = ones_row(1) * 1.0 * Units.deg
```

Comment:
This is to tell what angle the vehicle is at for debugging when unpacking an packing. Changed to match prior one.